randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -822,30 +822,47 @@ Extra - asset_id: collision_camera_011
   * Extra - left_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Dock to Map Station Access (dooremmy_002)
+  > Dock to Map Station Access (Lower)
       Trivial
+  > Dock to EMMI Zone Exit West
+      All of the following:
+          # If you start in the room direclty below this one, you can carry a speed boost all the way to here
+          Morph Ball and Speed Booster and Speedbooster Conservation (Beginner) and Shoot Plasma Beam
 
-> Dock to Map Station Access (dooremmy_002); Heals? False
+> Dock to Map Station Access (Lower); Heals? False
   * Layers: default
-  * EMMI Door to Map Station Access/Dock to EMMI Zone Exit Southeast (dooremmy_002)
+  * EMMI Door to Map Station Access/Dock to EMMI Zone Exit Southeast (Lower)
   * Extra - actor_name: dooremmy_002
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Door to Blue EMMI Introduction
       Trivial
-  > Dock to Map Station Access (dooremmy_003)
+  > Dock to Map Station Access (Upper)
       Any of the following:
-          Grapple Beam or Use Spin Boost
+          Grapple Beam or Space Jump or Lay Cross Comb
+          Slide and Slide Jump (Beginner) and Use Spin Boost
+          Flash Shift and Walljump (Intermediate)
           Gravity Suit and Simple IBJ
+          All of the following:
+              # This room allows for a bomb jump out of the water without a WBJ
+              Bomb and Morph Ball and Infinite Bomb Jump (Intermediate)
+          All of the following:
+              # If you start in the room directly below this one, you can carry a speed boost all the way here
+              Morph Ball and Speed Booster and Speedbooster Conservation (Advanced) and Shoot Plasma Beam
```

Comment:
Same comment here about needing the plasma beam requirement

Reply:
This door I believe should still have the plasma requirement since you are not guaranteed to come from the plasma door. All of Ghavoran can be completed without Plasma if you have Ice Missiles to go to Spin Boost Tower

Reply:
Gotcha. best to keep the plasma requirement then.